atlasdb-console/src/main/groovy/com/palantir/atlasdb/console/module/AtlasCoreModule.groovy

```diff
@@ -135,10 +136,12 @@ class AtlasCoreModule implements AtlasConsoleModule {
     }
 
     public connect(String yamlFilePath = null) {
```

Comment:
Looks like we should be able to consolidate the logic between these two 'connect / config load' methods.  A simple solution might be to have the config object have static method for loading from yaml.